lib/streamlit/runtime/scriptrunner/magic.py

```diff
@@ -15,10 +15,14 @@
 from __future__ import annotations
 
 import ast
+import sys
 from typing import Any, Final
 
 from streamlit import config
 
+MATCH_SYNTAX_AVAILABLE = sys.version_info >= (3, 10)
+TRY_STAR_SYNTAX_AVAILABLE = sys.version_info >= (3, 11)
```

Comment:
minor thing: maybe change the `TRY_STAR_SYNTAX_AVAILABLE` constant name to `EXCEPT_STAR_SYNTAX_AVAILABLE` to be aligned with the statement name.

Reply:
Thanks!
It makes sense, but fyi I named it after the `ast.TryStar` object.
Also, if we focus on the availability of the specific syntax, `EXCEPTION_GROUP_SYNTAX_AVAILABLE` might be another option.
What do you thing is the best?

Reply:
Yes, both options work for me; I think it can remain as is. 👍